support-frontend/assets/components/subscriptionCheckouts/orderSummary/orderSummary.jsx

```diff
@@ -22,12 +22,10 @@ type PropTypes = {
 
 
 function OrderSummary(props: PropTypes) {
```

Comment:
Same point about this component, it looks very specific to DS, will it be generalised at some point? (I realise that this was added before this PR)

Reply:
Yes - I'll move both. We can generalise it when needed.